src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  width: 100px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
You've used `display: block;` for `.stars__star`. Block-level elements start on a new line and take up the full width available. In your case, you've already set the width and height of the star, so you don't need to make it a block-level element. You can remove this property.